### PR TITLE
chore: Update page to point to the Terraform modules catalog on cloud.google.com

### PR DESCRIPTION
**Pull request by kaariger**

### CODE REVIEW

File: docs/meta/terraform.md.tmpl [hunk 1]
```diff
@@ -1,19 +1,5 @@
 <!--- This file is autogenerated, edit the template in meta/ -->
```
Comment (morgante) [inline docs/meta/terraform.md.tmpl:1]: Instead of modifying this template, we should just stop auto-generating the `terraform.md` file.

**Review by morgante — CHANGES_REQUESTED** [review]

File: docs/meta/terraform.md.tmpl (hunk 1 above)
Comment (kaariger) [reply docs/meta/terraform.md.tmpl:1]: The generator currently does not auto run but has to be manually triggered. So this file should not be updated unless manually done by running the code.

I can remove that code although I thought leaving the code was ok for now.

Comment (bharathkkb) [reply docs/meta/terraform.md.tmpl:1]: @kaariger how are we generating https://g.co/dev/terraformfoundation? I assumed that was using this template?

Comment (kaariger) [reply docs/meta/terraform.md.tmpl:1]: > @kaariger how are we generating https://g.co/dev/terraformfoundation? I assumed that was using this template?

That is currently being manually maintained.

**Comment by morgante** [comment]
Replaced by #1078.

**Comment by kaariger** [comment]
closing this since its superseded by #1078

File: docs/meta/terraform.md.tmpl (hunk 1 above)
Comment (bharathkkb) [reply docs/meta/terraform.md.tmpl]: Looking at the source of https://g.co/dev/terraformfoundation it does look like the generated txt is slightly different but seems like something we should add to the template.

**Pull request closed without merging**